unison-hashing-v2/src/Unison/Hashing/V2/Term.hs

```diff
@@ -162,6 +162,8 @@ instance (Var v) => Hashable1 (TermF v a p) where
                         [tag 1, Hashable.Text (Text.pack s)]
                       B.Recorded (B.Resolve _ s) ->
                         [tag 2, Hashable.Text (Text.pack s)]
+                      B.Recorded (B.MissingResultPlaceholder _) ->
+                        [tag 3, Hashable.Text (Text.pack "_")]
```

Comment:
@runarorama This looks like a problem maybe. It's reusing a tag that's already used for another case (see below).

Reply:
oh I see. Subtle!

Reply:
You mean `TermApp`? I would think it's ok because `MissingResultPlaceholder` would produce `1 : 3 : ...`, whereas `TermApp` would just produce `3 : ...`